styles/style.css

```diff
@@ -0,0 +1,92 @@
+  /**
+    Used only in the nav bar
+    */
+
+    .navbar {
+      border: 0;
+      border-radius: 0;
+      margin-bottom: 0;
+      font-size: 12px;
+      letter-spacing: 2px;
+      padding-top: 0px;
+      padding-bottom: 0px;
+    }
+
+    .navbar-nav  li a:hover {
+      color: #1abc9c !important;
+    }
+
+/**
+  Skills section
+  */
+  .progress {
+    position: relative;
+    height: 25px;
+    max-width: 1000px;
+  }
+
+  .progress > .progress-type {
+    position: absolute;
+    left: 0px;
+    font-weight: 800;
+    padding: 3px 30px 2px 10px;
+    color: rgb(0, 0, 0);
+  }
+
+  .progress > .progress-completed {
+    position: absolute;
+    right: 0px;
+    font-weight: 800;
+    padding: 3px 10px 2px;
+    color:black;
+  }
+
+  .progress-bar-css{
+    background-color:rgb(255, 255, 153) !important;
+  }
+
+  .progress-bar-human{
+    background-color: rgb(51, 208, 178) !important;
+  }
+
+
+  /**
+    Contact section
+  */
+
+  #contact{
+    background-color: white;
+    color:black;
+    text-align: center;
+    padding-bottom: 50px;
+    padding-top:50px;
+  }
+
+    #linkedin{
+      color:rgb(66, 133, 243);
```

Comment:
According to a quick Google search https://www.google.com/search?q=linkedin+logo+color+code&rlz=1C5CHFA_enUS724US727&oq=linkedin+logo+color+code&aqs=chrome..69i57.3586j0j1&sourceid=chrome&ie=UTF-8

LinkedIn's color is #0077B5, let's use that here instead